### PR TITLE
chore: update rustyline to 7.0.0

### PR DESCRIPTION
**Pull request by caspervonb**
Updates rustyline to 7.0.0 which introduces some new actions (key bindings), resolves, or at-least partially resolves https://github.com/denoland/deno/issues/8454

### CODE REVIEW

**Comment by caspervonb** [comment]
Had to bump a couple of transitives as-well.

File: Cargo.lock [hunk 1]
```diff
@@ -3235,6 +3327,30 @@ version = "0.2.68"
...
+
+[[package]]
+name = "wasm-bindgen-test-macro"
+version = "0.3.18"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "e8fb9c67be7439ee8ab1b7db502a49c05e51e2835b66796c705134d9b8e1a585"
+dependencies = [
+ "proc-macro2 1.0.24",
+ "quote 1.0.7",
+]
```
Comment (bartlomieju) [inline Cargo.lock:3352]: Where do these deps come from?

File: Cargo.lock [hunk 2]
```diff
@@ -3264,6 +3380,15 @@ dependencies = [
...
 ]
 
+[[package]]
+name = "webpki-roots"
+version = "0.20.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "0f20dea7535251981a9670857150d571846545088359b28e4951d350bdaf179f"
+dependencies = [
+ "webpki",
+]
```
Comment (bartlomieju) [inline Cargo.lock:3390]: Ditto, it's surprising this is being updated

File: Cargo.lock (hunk 1 above)
Comment (caspervonb) [reply Cargo.lock:3352]: Quick grep of the Cargo.lock file, they're from reqwest

File: Cargo.lock [hunk 3]
```diff
@@ -2037,17 +2095,18 @@ dependencies = [
...
  "serde",
- "serde_urlencoded",
- "tokio 0.2.22",
+ "serde_urlencoded 0.7.0",
+ "tokio 0.2.23",
  "tokio-rustls",
  "url",
  "wasm-bindgen",
  "wasm-bindgen-futures",
+ "wasm-bindgen-test",
```
Comment (caspervonb) [inline Cargo.lock:2107]: Here.

File: Cargo.lock (hunk 1 above)
Comment (magurotuna) [reply Cargo.lock:3352]: Maybe that's related to https://github.com/seanmonstar/reqwest/pull/1093

Comment (bartlomieju) [reply Cargo.lock:3352]: It's strange that `reqwest` gets upgraded anyway. Is that intentional?

Comment (caspervonb) [reply Cargo.lock:3352]: Not really, just merged with master and ran cargo update 🤔

Comment (bartlomieju) [reply Cargo.lock:3352]: > Not really, just merged with master and ran cargo update 🤔

Bad idea 😅  I suggest upgrading manually only crates you're interested in. @caspervonb could you revert non-rustyline related crate updates?

File: Cargo.lock (hunk 2 above)
Comment (caspervonb) [reply Cargo.lock]: Thanks @bartlomieju 🙃

**Review by bartlomieju — APPROVED** [review]
LGTM

**Pull request merged**